pwem/protocols/protocol_align_movies.py

```diff
@@ -563,6 +563,9 @@ def averageMovie(self, movie, inputFn, outputMicFn, binFactor=1, roi=None,
         if gain is not None:
             args += ' --gain ' + gain
 
+        if outxmd is not None:
```

Comment:
What happens if outxmd is not passed?

Reply:
If the outxmd filename is not passed, the function behaves as before these changes: the outxmd name will be the default (out.xmd if I recall correctly), which can be problematic when running in parallel.

Reply:
That is why y was asking....does is maybe appears at PROJECT_HOME?

I was thinking to have a default behaviour more convenient, like working in the protocol/tmp path and maybe having a timestamp based name to avoid collisions or something based on the move name?